docs/get-started/getting-started-with-rapids-accelerator-on-databricks.md

```diff
@@ -6,48 +6,52 @@ parent: Getting-Started
 ---
 
 # Getting started with RAPIDS Accelerator on Databricks
-This guide will run through how to set up the RAPIDS Accelerator for Apache Spark 3.0 on Databricks. At the end of this guide, the reader will be able to run a sample Apache Spark application that runs on NVIDIA GPUs on Databricks.
+This guide will run through how to set up the RAPIDS Accelerator for Apache Spark 3.0 on Databricks.  At the end of this guide, the reader will be able to run a sample Apache Spark application that runs on NVIDIA GPUs on Databricks.
 
 ## Prerequisites
-* Apache Spark 3.0 running in DataBricks Runtime 7.0 ML with GPU 
+    * Apache Spark 3.0 running in DataBricks Runtime 7.0 ML with GPU 
     * AWS: 7.0 ML (includes Apache Spark 3.0.0, GPU, Scala 2.12)
     * Azure: 7.0 ML (GPU, Scala 2.12, Spark 3.0.0)
 
 The number of GPUs per node dictates the number of Spark executors that can run in that node.
 
 ## Start a Databricks Cluster
 Create a Databricks cluster by going to Clusters, then clicking “+ Create Cluster”. Ensure the cluster meets the prerequisites above by configuring it as follows:
-1. On AWS, make sure to use 7.0 ML (includes Apache Spark 3.0.0, GPU, Scala 2.12), or for Azure, choose 7.0 ML (GPU, Scala 2.12, Spark 3.0.0).
+1. On AWS, make sure to use 7.0 ML (GPU, Scala 2.12, Spark 3.0.0), or for Azure, choose 7.0 ML (GPU, Scala 2.12, Spark 3.0.0).
 2. Under Autopilot Options, disable auto scaling.
 3. Choose the number of workers that matches the number of GPUs you want to use.
-4. Select a worker type. On AWS, use nodes with 1 GPU each such as p3.xlarge or g4dn.xlarge. p2 nodes do not meet the architecture requirements for the Spark worker (although they can be used for the driver node). For Azure, choose GPU nodes such as Standard_NC6s_v3
+4. Select a worker type.  On AWS, use nodes with 1 GPU each such as `p3.xlarge` or `g4dn.xlarge`.  p2 nodes do not meet the architecture requirements for the Spark worker (although they can be used for the driver node).  For Azure, choose GPU nodes such as Standard_NC6s_v3. 
 5. Select the driver type. Generally this can be set to be the same as the worker.
+6. Start the cluster
 
-## Advance Cluster Configuration
+## Advanced Cluster Configuration

```

Comment:
Could we add a section at the end to enable log delivery in databricks. Spark log in databricks will be destroyed with the cluster shutdown. Hence it is useful to provide a pointer to https://docs.databricks.com/clusters/configure.html#cluster-log-delivery-1, so that the log can be saved to a cloud storage.

Reply:
Dong, this comment is for internal users. We can discuss an internal page for these details.

Reply:
the log will be helpful for debugging and performance tuning, I know this is not a Nvidia problem, but nice to point out.

Reply:
I added a new Hints section talking about storing logs.  Can you review @mengdong ?

Reply:
thx!